packages/space-opera/src/components/mobile_view/utils.ts

```diff
@@ -78,15 +78,30 @@ export async function post(content: Blob|string, url: string) {
   }
 }
 
+// Aborting the get request as opposed to failing will keep the site pipe
+// running smoother.
+// https://dmitripavlutin.com/timeout-fetch-request/#2-timeout-a-fetch-request
+export async function getWithTimeout(url: string) {
```

Comment:
aborting pings (instead of failing hard after their internal 60 seconds delay) seems to speed up get/posts requests